### PR TITLE
Support display interface v1.2

### PR DESCRIPTION
**Pull request by shshshl09**

### CODE REVIEW

File: app/src/main/java/com/skt/nugu/sampleapp/template/FragmentTemplateRenderer.kt [hunk 1]
```diff
@@ -112,6 +111,29 @@ class FragmentTemplateRenderer(fragmentManager: FragmentManager, @IdRes private
...
+            val fragment = findFragmentByTemplateId(fragmentManager, templateId)
+            if(fragment is TemplateFragment) {
+                result = fragment.controlFocus(direction)
+                countDownLatch.countDown()
+            } else {
+                countDownLatch.countDown()
+                return@post // false
+            }
+        }
+
```
Comment (shshshl09) [inline app/src/main/java/com/skt/nugu/sampleapp/template/FragmentTemplateRenderer.kt]: countDownLatch's wait missed

File: nugu-agent/src/main/java/com/skt/nugu/sdk/agent/display/ControlFocusDirectiveHandler.kt [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            info.result.setFailed("[executeHandleControlFocusDirective] invalid payload")
+            removeDirective(info.directive.getMessageId())
+            return
+        }
+
+        if(controller.controlFocus(payload.playServiceId, payload.direction)) {
+            sendControlFocusEvent(info.directive.payload, "$NAME_CONTROL_FOCUS$NAME_SUCCEEDED")
+        } else {
+            sendControlFocusEvent(info.directive.payload, "$NAME_CONTROL_FOCUS$NAME_FAILED")
+        }
```
Comment (shshshl09) [inline nugu-agent/src/main/java/com/skt/nugu/sdk/agent/display/ControlFocusDirectiveHandler.kt]: missed success handling.

File: nugu-agent/src/main/java/com/skt/nugu/sdk/agent/DefaultDisplayAgent.kt [hunk 3]
```diff
@@ -325,53 +350,61 @@ class DefaultDisplayAgent(
...
+        }
+
+        val currentRenderedInfo = currentInfo ?: return
+        val currentPlayServiceId = currentRenderedInfo.payload.playServiceId
+        if (currentPlayServiceId.isNullOrBlank()) {
+            Logger.w(
+                TAG,
+                "[executeHandleCloseDirective] (Close) no current info (maybe already closed)."
+            )
+            sendCloseSucceeded(info, closePayload.playServiceId)
```
Comment (tommy5oo) [inline nugu-agent/src/main/java/com/skt/nugu/sdk/agent/DefaultDisplayAgent.kt]: Is it succeeded?

Comment (shshshl09) [reply nugu-agent/src/main/java/com/skt/nugu/sdk/agent/DefaultDisplayAgent.kt]: I agree

**Pull request merged**